dashboard/test/ui/features/step_definitions/steps.rb

```diff
@@ -143,6 +143,11 @@ def individual_steps(steps)
   wait_until {@browser.execute_script("return $(#{selector.dump}).text();").include? text}
 end
 
+When /^I wait until (?:element )?"([^"]*)" does not (?:have|contain) text "([^"]*)"$/ do |selector, text|
+  element_text = wait_until {@browser.execute_script("return $(#{selector.dump}).text();")}
+  !element_text.include? text
```

Comment:
I don't think this step behaves as advertised - it will rarely wait, since your `include?` check is outside the `wait_until` block.  Try this?
```rb
wait_short_until do
  element_text = @browser.execute_script("return $(#{selector.dump}).text();")
  !element_text.include? text
end
```

Also suggesting `wait_short_until` - 30 second timeout instead of two minutes.